rules/rules.libsonnet

```diff
@@ -373,19 +419,19 @@
             ||| % $._config,
           },
           {
-              record: 'node:node_inodes_free:',
-              expr: |||
+            record: 'node:node_inodes_free:',
```

Comment:
These just look like whitespace changes, right?

Reply:
According to `?w=1`, yes these are all just whitespace changes.